client/daemon.go

```diff
@@ -267,73 +266,24 @@ type Keys interface {
 
 // KeysFile is an implementation of Keys based on the file system for the register file.
 type KeysFile struct {
-	fn string
-	fd int
+	fn       string
```

Comment:
Might be better to just have a struct in the flock_<os> files and have the fd and lockfile be used there since they aren't used anywhere else in the code. That way if we needed to extend what they used later it'd be easier to understand. (also could potentially have the flock and funlock functions be part of that struct)